components/header/TransactionHeader.vue

```diff
@@ -182,6 +182,12 @@ export default {
     dateCompleted() {
       return new Date(this.timestamp * 1000).toString();
     },
+    effectiveToId() {
+      if (Array.isArray(this.toId) && (this.toId.length === 1)) {
+        return this.toId[0];
```

Comment:
for `this.toId.length > 1` should not return array directly, if `this.toId.length > 1` is an error case, should instant return empty string